sdk/core/azure-core/test/e2e/azure_core_with_curl_bodyStream.cpp

```diff
@@ -1,192 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// SPDX-License-Identifier: MIT
-
-/**
- * @brief Simulates customer application that is linked with azure-core and azure-transport-curl
- * with Stream Body
- *
- */
-
-#include <azure/core/http/curl/curl.hpp>
-#include <azure/core/http/http.hpp>
-#include <azure/core/http/pipeline.hpp>
-
-#include <array>
-#include <iostream>
-#include <memory>
-#include <vector>
-
-using namespace Azure::Core;
-using namespace Azure::Core::Http;
-using namespace std;
-
-// For bodyBuffer
-constexpr auto BufferSize = 50;
-std::vector<uint8_t> buffer(BufferSize);
-
-// For StreamBody
-constexpr auto StreamSize = 1024; // 100 MB
-std::array<uint8_t, StreamSize> bufferStream;
-
-void doGetRequest(Context const& context, HttpPipeline& pipeline);
-void doNoPathGetRequest(Context const& context, HttpPipeline& pipeline);
-void doPutRequest(Context const& context, HttpPipeline& pipeline);
-void doPutStreamRequest(Context const& context, HttpPipeline& pipeline);
-void printStream(Azure::Core::Context const& context, std::unique_ptr<Http::RawResponse> response);
-
-int main()
```

Comment:
Were all these passing at some point? Are we losing any test coverage that our existing unit tests don't cover? If so, we should extract parts of these tests into our unit tests if possible.

Reply:
They were!  But it never got plugged in to CI/Live tests.

These tests were born as an intent to share developing and debugging tools from me to other developers. 
It was intended to become, at some point, the samples for end-users which they could copy-paste to run.

However, it makes more sense to have those samples in the service's directories and not from Core.

It became harder and harder to maintain at the end, so, I will just prefer to remove and move it to some personal fork or something :)